src/main/java/org/sagebionetworks/template/repo/WebACLBuilderImpl.java

```diff
@@ -71,9 +71,11 @@ public void buildWebACL(List<String> environmentNames) {
 		String stackName = createStackName();
 		this.logger.info("Template for stack: " + stackName);
 		this.logger.info(resultJSON);
+		boolean enableTerminationProtection = ("prod".equals(config.getProperty(PROPERTY_KEY_STACK)));
```

Comment:
We can put the "prod" value in the constants, might be worth creating a utility method to check